package.json

```diff
@@ -48,90 +48,90 @@
     ]
   },
   "dependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
     "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "bcrypt": "^3.0.7",
-    "body-parser": "^1.18.3",
+    "@babel/polyfill": "^7.10.1",
+    "@babel/preset-env": "^7.10.1",
+    "@babel/preset-react": "^7.10.1",
+    "bcrypt": "^4.0.1",
+    "body-parser": "^1.19.0",
     "cezerin2-client": "^0.38.0",
-    "chart.js": "^2.7.2",
-    "clean-webpack-plugin": "^0.1.19",
-    "concurrently": "^4.0.1",
-    "cookie-parser": "^1.4.3",
-    "css-loader": "^1.0.0",
-    "cssnano": "^4.1.0",
-    "esm": "^3.0.80",
-    "express": "^4.16.3",
-    "express-jwt": "^5.3.0",
-    "formidable": "^1.2.1",
+    "chart.js": "^2.9.3",
+    "clean-webpack-plugin": "^3.0.0",
```

Comment:
set to minor version. 3.0.0 breaks webpack by rendering.

Reply:
Sure about this? Because I have to edit webpack updates came with this...

Reply:
100% sure, we have tested before with the dependabot system cleanup pr's. It set up to this major version and the build of webpack fails on running: npm run build.

Reply:
I had to revert the merge because of it.

Reply:
Okay I change it... Thanks for details.